.github/workflows/etterlatte-tilbakestill-alle-prod.yaml

```diff
@@ -0,0 +1,91 @@
+name: etterlatte-tilbakestill-alle
```

Comment:
Må denne også slutte med `-prod` for å unngå navnekollisjon med eksisterende jobb, eller går det fint tro? 🤔

Reply:
Samme med `.deploy-main-prod.yaml` forsåvidt.

Reply:
stemmer, skal fikse